messages/descriptions/platform.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Platform description",
+    "description": "A description of a platform including metadata of the platform itself, available sensors and produced observations",
+    "required": [
+        "platform_id",
+        "metadata",
+        "sensors"
+    ],
+    "properties": {
+        "platform_id": {
+            "type": "string",
+            "title": "Platform id",
+            "desciption": "Id of the platform"
+        },
+        "metadata": {
+            "type": "object",
+            "title": "Metadata",
+            "description": "Metadata of this platform",
+            "required": [
+                "length",
+                "breadth"
```

Comment:
Perhaps we could have "visual_representation" instead of length and breadth. Then we could have, for example, a geojson polygon for the contour of the ship, a point for a person, a line for a snake, etc. Things like air_draft, draft, or depth could be added as properties.